pkg/shell/hosts_dialog.jsx

```diff
@@ -823,7 +831,7 @@ class ChangeAuth extends React.Component {
             {statement}
             <br />
             {(offer_login_password || offer_key_password) &&
-                <Form isHorizontal>
+                <Form isHorizontal onSubmit={prevent_default(callback)}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19018-20230627-115453-74cfaf23-fedora-38-devel-storage//Coverage/lcov/github-pr.diff.gcov.html)